src/Trie.mo

```diff
@@ -353,6 +480,24 @@ module {
   /// note: the `disj` operation generalizes this `merge`
   /// operation in various ways, and does not (in general) lose
   /// information; this operation is a simpler, special case.
+  ///
+  /// For a more detailed overview of how to use a Trie,
+  /// see the [User's Overview](#overview).
+  /// 
+  /// Example:
+  /// ```motoko include=initialize
+  /// trie := Trie.put(trie, key "hello", Text.equal, 42).0; 
+  /// trie := Trie.put(trie, key "bye", Text.equal, 42).0; 
+  /// // trie2 is a copy of trie
+  /// var trie2 = Trie.clone(trie); 
```

Comment:
The clone operation is actually just the identity since this a functional datastructure.
You could just write
/// var trie2 = trie1;